installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc

```diff
@@ -0,0 +1,8 @@
+[id="ipi-install-post-installation-configuration"]
+= Post-installation configuration
```

Comment:
I might add " installer-provisioned cluster" or something similar to the title, since we have a larger post-installation configuration section elsewhere